package.json

```diff
@@ -11,10 +11,11 @@
   },
   "main": "index.js",
   "scripts": {
-    "prepare": "npm prune && npm run build",
-    "build": "rm -rf lib && babel src --out-dir lib -s > /dev/null",
+    "prepare": "pwd && rm -rf ./lib",
+    "postinstall": "npm run build",
+    "build": "./node_modules/.bin/babel ./src --out-dir ./lib -s > /dev/null",
     "test": "eslint -c .eslintrc.json src test && npm run test:unit && npm run test:integration",
-    "test:unit": "nyc mocha --require babel-register --recursive ./test/unit",
+    "test:unit": "nyc mocha --require js:@babel/register --recursive ./test/unit",
```

Comment:
Mistake here, should be `nyc mocha --require @babel/register --recursive ./test/unit`

Reply:
Done